packages/fiori-elements-writer/test/common.ts

```diff
@@ -115,3 +119,39 @@ export const v2TemplateSettings: LROPSettings | WorklistSettings = {
         }
     }
 };
+
+export const projectChecks = async (
+    rootPath: string,
+    config: FioriElementsApp<unknown>,
+    debugFull = false
+): Promise<void> => {
+    if (debugFull && (config.appOptions?.typescript || config.appOptions?.eslint)) {
```

Comment:
Seems odd to have the `debugFull` condition wrapping the entire code block. Would be clearer to have this test at the point of call?